sample.env

```diff
@@ -0,0 +1,8 @@
+export HOST=0.0.0.0
+export PORT=1234
+export LOGINAPI=https://user:password@login.webmaker.org
+export COOKIE_SECRET=topSecretPasswordForEncryptingCookies
+export SECURE_COOKIES=true
```

Comment:
This should be defaulted to false, since the local server is probably running on http